src/delete-completed-tasks.js

```diff
@@ -0,0 +1,12 @@
+const deleteAllCompleted = (task) => {
+  const newArray = [];
+  for (let i = 0; i < task.length; i += 1) {
+    if (task[i].completed === false) {
+      newArray.push(task[i]);
+    }
+  }
+  task.length = 0;
+  task.push(...newArray);
+};
+
```

Comment:
- [ ] You did a great job by implementing the delete all completed feature, however, Kindly make sure you use the `filter` method as specified in the project requirement. In that way, your logic will be much more efficient with a good time complexity.

Hint: 
```jsx
arrayOfTasks.filter(tasks= > tasks !== task)
```

Reply:
@christianbiring1 hello, thank you for your review. I tried using filter() but it doesn't work as intended.
I tried this but for some reason, some elements in the list are duplicated. Could you help me?

`const deleteAllCompleted = (task) => {
  const newArray = task.filter(taskItem => !taskItem.completed);
  task.splice(0, task.length, ...newArray);
  console.log(task);
};`

Reply:
hello @christianbiring1 the filter() method is behaving in unpredictable ways. I cannot use it.